TreasureData/TreasureData.h

```diff
@@ -9,6 +9,25 @@
 #import <Foundation/Foundation.h>
 #import "TDClient.h"
 
+#if UIKIT_STRING_ENUMS
+typedef NSString * TDRequestOptionsKey NS_TYPED_ENUM;
+#else
+typedef NSString * TDRequestOptionsKey;
+#endif
+
+#if UIKIT_STRING_ENUMS
```

Comment:
Merge these two conditional macros for better readability?

Reply:
I was following Apple conventions. I think we should keep them separate. But I will separate TDRequestOptionsKey onto different header file